Tests/UIKitTests/UIImageExtensionsTests.swift

```diff
@@ -13,6 +13,27 @@ import XCTest
 import UIKit
 
 final class UIImageExtensionsTests: XCTestCase {
+    func testAverageColor() {
```

Comment:
Minor nitpick, but can you please add an empty space?
```suggestion

    func testAverageColor() {
```